content/blog/2021/announcing-keynote-speaker-sebastian.md

```diff
@@ -0,0 +1,28 @@
+Title:Announcing our second Keynote Speaker for PyCon India 2021 - Sebastián Ramírez
+Date: 2021-06-02 17:00
+Category: 2021
+Slug: keynote-announcement-sebastián-ramírez
```

Comment:
should we have "á" or "a" for slug?

Reply:
Let's keep all english common characters in url?
`keynote-announcement-sebastian-ramirez` ??

Reply:
I was thinking about it at the time of creating the PR, will make the change